.gitignore

```diff
@@ -34,6 +34,7 @@ nosetests.xml
 coverage.xml
 *.cover
 .hypothesis/
+result_images
```

Comment:
Running `check_figures_equal` in tests will save the generated Matplotlib figures inside this folder, so added it gitignore just in case. https://github.com/matplotlib/matplotlib/blob/27aaf0c378f767d5bcff4f2a65664485fba1f69b/.gitignore#L81